Sources/PenguinParallel/NonblockingThreadPool/NumberOperations.swift

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020 Penguin Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// Returns an array of all positive integers that are co-prime with `n`.
```

Comment:
```suggestion
/// Returns the positive integers that are coprime with `n`.
```

Reply:
Excellent. Thank you!

Reply:
Maybe also say they're in increasing order, if they are?